### PR TITLE
Implemented "crashes upload-missing-symbols" command

### PR DESCRIPTION
**Pull request by bryukov**
Implemented command according to the specification
crashes-upload-missing-symbols

### CODE REVIEW

File: src/commands/crashes/lib/symbols-uploading-helper.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+  private calculateMd5(buffer: Buffer): Promise<string> {
+    return new Promise<string>((resolve) => {
+      resolve(Crypto.createHash("md5").update(buffer).digest("base64"));
+    });
+  }
+
+  private async executeSymbolsUploadingBeginRequest(client: MobileCenterClient, app: DefaultApp): Promise<models.SymbolUploadBeginResponse> {
+    this.debug("Executing API request to get uploading URL");
+    let symbolUploadingBeginRequest = clientRequest<models.SymbolUploadBeginResponse>((cb) => client.symbols.postSymbolUpload(
```
Comment (christav) [inline src/commands/crashes/lib/symbols-uploading-helper.ts]: Why the separate variable and await call on line 69? Why not just await on this line here?

File: src/commands/crashes/upload-missing-symbols.ts [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+import * as JsZipHelper from "../../util/misc/jszip-helper";
+import * as _ from "lodash";
+import * as Os from "os";
+import * as ChildProcess from "child_process";
+
+const debug = require("debug")("mobile-center-cli:commands:apps:crashes:upload-missing-symbols");
+const mdfind = require("mdfind");
+const bplist = require("bplist");
+
+@help("Upload the missing crash symbols for the application")
```
Comment (christav) [inline src/commands/crashes/upload-missing-symbols.ts]: Add "(only from macOS)" to the help text.

Comment (christav) [inline src/commands/crashes/upload-missing-symbols.ts]: Also remove the "the" ... "Upload missing crash symbols..."

File: src/commands/crashes/lib/symbols-uploading-helper.ts (hunk 1 above)
Comment (bryukov) [inline src/commands/crashes/lib/symbols-uploading-helper.ts]: Fixed

File: src/commands/crashes/upload-missing-symbols.ts (hunk 2 above)
Comment (bryukov) [inline src/commands/crashes/upload-missing-symbols.ts]: Fixed

**Pull request merged**